### PR TITLE
Create income-verification.vbs

### PR DESCRIPTION
**Pull request by KallistaImdieke**
Here is the script that will pull what CP is receiving on her case with you along with all payments on all her cases. for when they call in wanting information regarding her cases for Tricap, any assistance programs, ect.

### CODE REVIEW

**Comment by KallistaImdieke** [comment]
@theVKC I also emailed you the word document that is used with this script and this case also needs the language added for the 3-6 mont and 1 year buttons.  

I think I may have forgotten to remove my redirect language to be able to pull the word document from my local computer let me know if i should fix this before or after you pull it in. 

thanks!

File: action/income-verification.vbs [hunk 1]
```diff
@@ -0,0 +1,291 @@
+'GATHERING STATS----------------------------------------------------------------------------------------------------
+name_of_script = "ACTIONS - CHILD SUPPORT INCOME VERIFICATION.vbs"
```
Comment (theVKC) [inline action/income-verification.vbs]: Must match the file name exactly.

File: action/income-verification.vbs [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    ELSE
+        FuncLib_URL = "C:\BZS-FuncLib\MASTER FUNCTIONS LIBRARY.vbs"
+        Set run_another_script_fso = CreateObject("Scripting.FileSystemObject")
+        Set fso_command = run_another_script_fso.OpenTextFile(FuncLib_URL)
+        text_from_the_other_script = fso_command.ReadAll
+        fso_command.Close
+        Execute text_from_the_other_script
+    END IF
+END IF
+
```
Comment (theVKC) [inline action/income-verification.vbs]: Same as the other one, can you add the changelog block?

File: action/income-verification.vbs [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+	child_list = child_list & child_name & VBnewline
+	Next
+
+'MCI Array 0 = MCINumber
+'MCI Array 1 = Child_F
+'MCI Array 2 = Child_M
+'MCI Array 3 = Child_L
+'MCI Array 4 = Child_S
+
+word_documents_folder_path = "E:\Kallistas Scripts\"
```
Comment (theVKC) [inline action/income-verification.vbs]: Can you delete this from the final pulled version?

**Comment by KallistaImdieke** [comment]
@theVKC  Let me know if you need anymore changes to this I am trying to figure out if I got all the ones you requested made!  Thanks!

**Comment by theVKC** [comment]
Nice job!

**Pull request merged**